db/migrate/20201116143058_add_last_name_to_user.rb

```diff
@@ -0,0 +1,5 @@
+class AddLastNameToUser < ActiveRecord::Migration[6.0]
+  def change
+    add_column :users, :last, :string
```

Comment:
I think you forgot the _name here

Reply:
Yes, j'ai corrigé le tir. last => last_name